tests/runtests.py

```diff
@@ -193,6 +193,14 @@ def test_cmd_heap_bins_fast(self):
         self.assertIn("Fastbins[idx=0, size=0x10]", res)
         return
 
+    def test_cmd_non_main_heap(self):
```

Comment:
could you call this something like `test_cmd_heap_bins_non_main_heap` and move it up so it's with the other heap tests?

Reply:
Ah yeah I didn't actually look where the code was, I only noticed that it ran before `hexdump` because of the name

Reply:
that was too long for the linter, think this one is good. force pushed to remove the extra commits